### DISCUSSION TITLE
Ad revenue share ?

### DISCUSSION THREAD

**Discussion by takhimi** [Poing-Studios/Godot-AdMob-Android-iOS#97] [General]
Hi
This is really good plugin or addon for godot. So I would like to confirm with you guys , does this addon or plugin implement some sort of ad revenue sharing of some sort ? For example during my cordova days few years ago, the cordova addon creator implement some sort of ad revenues sharing with user (it will shows the creator ads sometimes in user apps or games ). 

Does this addon do this kind of thing also ? Is it really completely free ?

**Comment by gumaciel** [comment]
Hello, this plugin just facilitates communication between Godot and AdMob, both for Android and iOS.

Could you give details of how "revenue sharing" works in Cordova? I believe this can exist on the AdMob platform: http://admob.google.com/ but i believe this would not apply in our plugin.

**Comment by takhimi** [comment]
Hi, the revenue sharing is actually implemented in the cordova plugin. Basically the creator will swap the ads with their own ads at some point and to avoid that you need to buy the "Pro" version. 

Anyway good to know that this is not implemented in this addon. Thank you so much for this addon.

**Reply by gumaciel** [reply]
Hi, as i know AdMob doesn't support this, sorry
